### PR TITLE
Dismiss reviews after pushing a new commit

### PR DESCRIPTION
**Pull request by mrunge**
to prevent accidental merges of non-reviewed
changes.

Related: https://github.com/Mergifyio/mergify-engine/issues/1860

### CODE REVIEW

File: .mergify.yml
Comment (jd) [inline .mergify.yml:27]:
```suggestion
    conditions: []
```

**Review by jd — CHANGES_REQUESTED** [review]

File: .mergify.yml
Comment (mrunge) [reply .mergify.yml]: Thanks, will do that

**Comment by mrunge** [comment]
Shouldn't the update also dismiss the review?

**Comment by jd** [comment]
It will, once this is merged.

**Comment by mrunge** [comment]
Okay, thank you Julien!

**Pull request merged**
